Sources/NIOTransportServices/NIOTSChannelOptions.swift

```diff
@@ -21,6 +22,19 @@ public struct NIOTSChannelOptions {
     public static let waitForActivity = NIOTSChannelOptions.Types.NIOTSWaitForActivityOption()
 
     public static let enablePeerToPeer = NIOTSChannelOptions.Types.NIOTSEnablePeerToPeerOption()
+    
+    public static let currentPath = NIOTSChannelOptions.Types.NIOTSCurrentPathOption()
+
+    @available(OSX 10.14, iOS 12.0, tvOS 12.0, watchOS 5.0, *)
```

Comment:
Nit: we don't need this annotation, the one on `NIOTSChannelOptions` is sufficient (it's a strict superset of this one).